src/tests/ftest/container/create.yaml

```diff
@@ -14,12 +14,8 @@ pool:
 uuids: !mux
    perfect:
      uuid:
-       - 0cf3086e-daa3-44a7-a5d4-cba9d639bf19
+       - VALID
```

Comment:
Hmm..Why "VALID" as uuid could pass?

Reply:
Both daos_cont_create() and dfs_cont_create() drop the support of passing uuid string. Now these two functions need a pointer of uuid_t. "VALID" means a valid pointer of uuid_t.